src/EditorFeatures/CSharpTest/Completion/CompletionProviders/TypeImportCompletionProviderTests.cs

```diff
@@ -33,11 +33,14 @@ internal override Type GetCompletionProviderType()
 
         private bool IsExpandedCompletion { get; set; } = true;
 
+        private bool DisallowAddingImports { get; set; }
```

Comment:
nit: i'm not a fan of negative booleans.  can we have this be `AllowAddingImports = true` and have it set to false by however needs that behavior?

Reply:
Will leave Disallow for now, we can refactor later if needed.